scripts/aws/cfn/microservice-dev-cfg.json

```diff
@@ -16,7 +16,7 @@
     "MemoryAmount": "4GB",
     "MaxAttachmentsCount": "15",
     "MaxRecipientsCount": "15",
-    "LollipopExpectedLcOriginalUrl": "^https://api-app.io.pagopa.it/api/v1/third-party-messages/[a-zA-Z0-9]{26}$",
+    "LollipopExpectedLcOriginalUrl": "^https://api-app.io.pagopa.it/api/v1/third-party-messages/[a-zA-Z0-9]{26}(:?/attachments/delivery/notifications/received/([A-Z]{4}-[A-Z]{4}-[A-Z]{4}-[0-9]{6}-[A-Z]-[0-9])/attachments/payment/(F24|PAGOPA)(\\?attachmentIdx=\\d+)?)?$",
```

Comment:
non mi sembra corretta... partire dalla regExp: `https://api-app.io.pagopa.it/api/v1/third-party-messages/[a-zA-Z0-9]{26}(:?/attachments/delivery/notifications/received/([A-Z]{4}-[A-Z]{4}-[A-Z]{4}-[0-9]{6}-[A-Z]-[0-9])/attachments/documents/[0-9])?` ed aggiungere anche la regola per accettare l'url: `https://api-app.io.pagopa.it/api/v1/third-party-messages/asdasd123qwasdasdasdvxvs23/attachments/delivery/notifications/received/ENZE-EUJU-UMGL-202306-H-1/attachments/payment/F24/?attachmentIdx=0`
Per la validazione utilizzare il test: **it.pagopa.pn.delivery.validation.ValidationsTest#regexTest**